flight/Modules/Stabilization/stabilization.c

```diff
@@ -420,9 +448,6 @@ static void stabilizationTask(void* parameters)
 						pids[PID_RATE_ROLL + i].iAccumulator = 0;
 					}
 
-					// The unscaled input (-1,1)
```

Comment:
worse, it breaks due to re-definition when it is in there multiple times. any suggestions?